lib/honeypot-captcha.rb

```diff
@@ -15,7 +15,7 @@ def honeypot_style_class
     end
 
     def protect_from_spam
-      head :ok if honeypot_fields.any? { |f,l| !params[f].blank? }
+      head :ok if honeypot_fields.any? { |f,l| !params[f].blank? || params[f].to_s.length > 0 }
```

Comment:
Perhaps this could be simplified to the one string value we allow: 

```suggestion
      head :ok if honeypot_fields.any? { |f, _| params[f].to_s != "" }
```